src/app/CommandHandler.cpp

```diff
@@ -482,7 +484,7 @@ CHIP_ERROR CommandHandler::AddStatusAndLogIfFailure(const ConcreteCommandPath &
                      ChipLogValueIMStatus(aStatus), aMessage);
     }
 
-    return AddStatus(aCommandPath, aStatus);
+    LogErrorOnFailure(AddStatus(aCommandPath, aStatus));
```

Comment:
I expect this is why you are getting noticeable flash increases on all platforms.  I would strongly suggest leaving this LogErrorOnFailure (which has the issues described in https://github.com/project-chip/connectedhomeip/issues/28532) to a separate PR.

Reply:
I added separate logging as it seems that this method is actually indended to not lose error statuses since it returns void.

Made a custom text to actually have full data, even though this will probably still have some slight size increase (I assume not too bad as this method is not inlined).

Reply:
> I added separate logging as it seems that this method is actually indended to not lose error statuses

This method is based on the reality that AddStatus in fact never fails....